tests/unit-test/test_serializer.c

```diff
@@ -560,6 +596,7 @@ int main(void) {
   RUN_TEST(test_serialize_ta_find_transactions_obj_by_tag);
   RUN_TEST(test_recv_mam_message_request_psk_deserialize);
   RUN_TEST(test_recv_mam_message_request_ntru_deserialize);
+  RUN_TEST(test_recv_mam_message_response_serialize);
```

Comment:
Where is the unit test of `test_recv_mam_message_response_deserialize`?

Reply:
This serialization function serves only for `/tests`, so I didn't add a test for it.